Code/GraphMol/MolInterchange/molinterchange_catch.cpp

```diff
@@ -422,4 +422,38 @@ TEST_CASE("github #5923: add more error checking to substance groups") {
                       ValueErrorException);
     }
   }
-}
\ No newline at end of file
+}
+
+TEST_CASE("github #8031: make sure floating point coords are not truncated") {
```

Comment:
```suggestion
TEST_CASE("github #8031: make sure floating point coords are not incorrectly truncated") {
```

Reply:
Done. I also fixed a typo (is not get => is not) in a comment.